hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -110,6 +110,116 @@ <h2>Space Statistics</h2>
     </tbody>
 </table>
 
+<h2>Pipeline Statistics</h2>
+<table class="table table-bordered table-striped">
+    <tbody>
+    <tr>
+        <th>Pipeline State</th>
+        <th>Size</th>
+    </tr>
+    <tr>
+        <td>Closed</td>
+        <td>{{statistics.pipelines.closed}}</td>
+    </tr>
+    <tr>
+        <td>Allocated</td>
+        <td>{{statistics.pipelines.allocated}}</td>
+    </tr>
+    <tr>
+        <td>Open</td>
+        <td>{{statistics.pipelines.open}}</td>
+    </tr>
+    <tr>
+        <td>Dormant</td>
+        <td>{{statistics.pipelines.dormant}}</td>
+    </tr>
+    </tbody>
+</table>
+
+<h2>Container Lifecycle Statistics</h2>
```

Comment:
I think the tables can be put under one `h2` header.

```
<h2>Container Statistics</h2>
<table class="table table-bordered table-striped">
    <tbody>
    <tr>
        <th>Operational State</th>
        <th>Size</th>
    </tr>
    ...
</table>
<table class="table table-bordered table-striped">
    <tbody>
    <tr>
        <th>Health</th>
        <th>Size</th>
    </tr>
    ...
</table>
```

Reply:
Thanks @ivandika3 .
I have updated it.
Here is the latest UI:
![image](https://github.com/user-attachments/assets/cd2a5ad4-cf2f-4066-937a-22b3c2e189ae)

And I have updated the description of the PR.

Reply:
Thanks for the update, is it possible to have a white space in between the table?

Reply:
@ivandika3 , this is the UI style we expect, right?
![image](https://github.com/user-attachments/assets/9db32338-2af6-4d64-95ca-a9e0b83b4bd4)

Reply:
Yes, that's what I expected, but the previous screenshot did not have a white space between the table.

Reply:
Thanks @ivandika3 .
I have updated it.
And I have updated the description of this PR.

Reply:
Thanks it looks good.